spec/controllers/no_draft_page_open_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+feature "Draft vaccination controller errors", type: :feature do
+  [
+    "/draft-vaccination-record/batch",
+    "/draft-vaccination-record/date_and_time",
```

Comment:
This path isn't valid, the `date_and_time` step [is mapped to `date-and-time`](https://github.com/nhsuk/manage-vaccinations-in-schools/blob/8b814db86a61f49259c9a2ec99f5ac64f9bef3f0/config/locales/en.yml#L714).

But this got me thinking, do we want to make this check somehow lower priority, so users see the normal 404 error page if the URL itself doesn't refer to a real step?